.github/workflows/e2e-kuttl-test.yaml

```
@@ -25,43 +25,34 @@ jobs:
-        run: |
-          kind create cluster --config ./kind-config.yaml
-          kubectl config use-context kind-elasti-e2e
-          kubectl cluster-info
-          kubectl get nodes
+          version: 'v3.12.3'
 
       - name: Build and push images
         working-directory: ./tests/e2e
         run: |
           make build-images
+      
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)